assets/src/features/leaving-page/components/QuestionnairePageLayout.tsx

```diff
@@ -4,17 +4,19 @@ import BlockingScreen from "../../shared/components/BlockingScreen";
 
 import Navbar from "./Navbar";
 import useSmartphoneViewport from "../../shared/hooks/useSmartphoneViewport";
+import useDynamicHeightResize from "../../shared/hooks/useDynamicHeightResize";
 
 const QuestionnairePageLayout: React.FC<PropsWithChildren> = ({ children }) => {
   const { isSmartphone, isHorizontal } = useSmartphoneViewport();
   const shouldBlockScreen = isSmartphone && isHorizontal;
 
+  useDynamicHeightResize();
   return (
     <>
       {shouldBlockScreen && <BlockingScreen message="Turn your screen to join the call." />}
       <div
         className={clsx(
-          "home-page h-screen w-full",
+          "home-page h-inner-height w-full",
```

Comment:
I'm not sure which place is best, but you should add a `TODO` note suggesting using `dvh` units when support is more common in browsers.

Reply:
I've added them to the hook itself and a reference in the `main.css`.